Point_set_processing_3/test/Point_set_processing_3/test_poisson_eliminate.cpp

```diff
@@ -0,0 +1,91 @@
+#include <vector>
+#include <iostream>
+#include <fstream>
+#include <filesystem>
+#include <string>
+
+#include <CGAL/Simple_cartesian.h>
+#include <CGAL/Exact_predicates_exact_constructions_kernel.h>
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+#include <CGAL/poisson_eliminate.h>
```

Comment:
In tests I suggest to `#include` in another order. First the files we really want to test, so that we are sure that they include everything that is needed.